### DISCUSSION TITLE
border-image support

### DISCUSSION THREAD

**Discussion by ewan-dgc** [cssbattle/forum#22] [Q&A] [answered]
Heyya,

Is border-image supported? i cant seem to get it to work

**Comment by chinchang** [comment]
It should work unless you are using images with it :)

**Comment by SelenIT** [comment, marked as answer]
It works with gradients. E.g., `<style>*>*{background:#62374e;margin:50;border-image:conic-gradient(#fdc57b 0 0)99%fill/53q` is a working solution for [target #2](https://cssbattle.dev/play/2). But I admit that behavior of gradients in `border-image` might be not very intuitive.
